Marlin/servo.cpp

```diff
@@ -78,7 +78,7 @@ static inline void handle_interrupts(timer16_Sequence_t timer, volatile uint16_t
   if (Channel[timer] < 0)
     *TCNTn = 0; // channel set to -1 indicated that refresh interval completed so reset the timer
   else {
-    if (SERVO_INDEX(timer,Channel[timer]) < ServoCount && SERVO(timer,Channel[timer]).Pin.isActive)
+    if (SERVO_INDEX(timer,Channel[timer]) < ServoCount)
       digitalWrite( SERVO(timer,Channel[timer]).Pin.nbr,LOW); // pulse this channel low if activated
```

Comment:
Correct, it is just a flag to signify that the specific servo is active. The problem was that when using the servo deactivation delay, the servo could be made inactive after the delay period, but while the line was still high, and without this fix the line would then remain high, causing all sorts of bother. 
This fixes one specific problem, but I the problem of continuous jitter i see reported must have another cause.